### ISSUE TITLE
Consider to publish Mac package or provide better documentation on how to compile

### ISSUE THREAD

**Issue by LightMoon** [MatsuriDayo/nekoray#442]
I am not a developer but I gave it go and tried to compile Mac version on my own which I enconter to following error.

<img width="738" alt="Screen Shot 2023-03-14 at 10 52 44 am" src="https://user-images.githubusercontent.com/1636991/224857239-5ba46a49-946d-4f9c-bd0d-73bba92fec26.png">

The following line in the documentation is not clear for non developer people, please clarify.

Install the Qt 5.15.xcompilation environment and set relevant environment variables so that CMake cannot find Qt. ( qtbase qtsvg qttools)

Thanks!

**Comment by mrAlishah** [comment]
It's not good idea to stop compiling Macos dmg. please keep going it.
I went into trouble too.
will Appreciate to change your choice.

**Comment by aaaamirabbas** [comment]
I also agree and have a serious problem.
Please create the correct documentation for compiling.
Or **re-release the Mac version like Windows and Linux**.

Thanks

**Comment by aaaamirabbas** [comment]
**Unofficial released: https://github.com/MatsuriDayo/nekoray/issues/445**

**Comment by Stevemoretz** [comment]
Yeah, it doesn't matter that it wasn't tested at least it worked!
You could just mention those builds are untested but still let github build them, or at least point us to a commit that handled that build so we can fork it.
